Makefile

```
@@ -0,0 +1,287 @@
+.PHONY: build install clean test run lint coverage bench release-dry release
+
+# Variables
+BINARY_NAME=cloud-update
+BUILD_DIR=build
+INSTALL_DIR=/opt/cloud-update
+SERVICE_FILE=src/init/systemd/cloud-update.service
+CONFIG_DIR=/etc/cloud-update
+VERSION=$(shell git describe --tags --always --dirty)
+COMMIT=$(shell git rev-parse --short HEAD)
+DATE=$(shell date -u +%Y-%m-%dT%H:%M:%SZ)
+LDFLAGS=-ldflags "-s -w -X cloud-update/src/internal/version.Version=$(VERSION) -X cloud-update/src/internal/version.Commit=$(COMMIT) -X cloud-update/src/internal/version.Date=$(DATE)"
```

checkmake (0.2.2)

[warning] Variable "DATE" possibly contains a timestamp and should be simply expanded.

(timestampexpanded)